File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieTrailer.java
package br.com.popularmovies.services.movieService.response;

import com.squareup.moshi.Json;

public class MovieTrailer {
    @Json(name = "id")
    private String id;

    @Json(name = "iso_639_1")
    private String iso6391;

    @Json(name = "iso_3166_1")
    private String iso31661;

    @Json(name = "key")
    private String key;

    @Json(name = "name")
    private String name;

    @Json(name = "site")
    private String site;

    @Json(name = "size")
    private String size;

    @Json(name = "type")
    private String type;

    public String getName() {
        return name;
    }

    public String getKey() {
        return key;
    }
}

File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieTrailer.kt
package br.com.popularmovies.services.movieService.response

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

@JsonClass(generateAdapter = true)
@Parcelize
data class MovieTrailer(@Json(name = "id")
                        val id: String,
                        @Json(name = "iso_639_1")
                        val iso6391: String,
                        @Json(name = "iso_3166_1")
                        val iso31661: String,
                        @Json(name = "key")
                        val key: String,
                        @Json(name = "name")
                        val name: String,
                        @Json(name = "site")
                        val site: String,
                        @Json(name = "size")
                        val size: String,
                        @Json(name = "type")
                        val type: String) : Parcelable

